frontend/src/components/header/Header.tsx

```diff
@@ -30,7 +34,10 @@ const Header: React.FC<HeaderProps> = ({isLogged}: HeaderProps) => {
         setAnchorElUser(null);
         const component = event.currentTarget.textContent ?? '';
         const route = findRouteByPath(component);
-        if (route) {
+        if (component === Titles.LOG_OUT) {
```

Comment:
Why do you compare react component to title ?

Reply:
i compare component name with constant, because `Log out` doesn't have separate page. Maybe i should rename `component` to make it more obvious that I'm comparing